optional-operators-subscribe.yml

```diff
@@ -8,3 +8,7 @@
     - name: "Run optional operators subscribe step"
       include_role:
         name: optional-operators-subscribe
+
+    - name: "Run the operator scorecard tests step"
```

Comment:
We should probably either rename this playbook to make it clear that it's meant to be run by external tests in DCI, or make running this role optional based on an input parameter. The name `optional-operators-subscripte.yml` for this playbook looks a bit confusing otherwise.

Reply:
Would it be better if I made a separate playbook?  I combined them because the scorecard test has to be run after the subscribe test, but I don't see anything wrong with two separate playbooks

Reply:
That would be fine.  I was planning that it would be a different playbook like I show here:

https://github.com/redhat-cip/ansible-role-dci-cvp/blob/master/tasks/run_cvp_operator.yml